docs/lib/redis.md

```diff
@@ -8,19 +8,19 @@ To leverage this client you can setup your Django cache like:
         "default": {
             "BACKEND": "django_redis.cache.RedisCache",
             # Note, the location is not really used but we parse it in the client to get settings like host/port/username/etc.
-            "LOCATION": f"redis://{os.environ.get('REDIS_USER')}:{os.environ.get('REDIS_PASS')}@{os.environ.get('REDIS_HOST')}:{os.environ.get('REDIS_PORT')}",
+            "LOCATION": f"<redis, rediss, file or unix schemed URL>",
```

Comment:
nit: typo `rediss`

Reply:
I know this looks weird but `rediss` is actually a legitimate scheme; Its like `http` vs `https` but it looks weird because redis ends with `s` already. So its just specify a tls enabled connection to redis.

Reply:
oh! sounds good, sorry, just knew about this.